bundle/default-bundle/src/main/resources/application.properties

```diff
@@ -11,3 +11,7 @@ camunda.operate.client.url=http://localhost:8081
 camunda.operate.client.username=demo
 camunda.operate.client.password=demo
 camunda.connector.webhook.enabled=true
+
+management.server.port=9080
```

Comment:
FYI: we will also need to update Helm charts: https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/templates/connectors/deployment.yaml#L131

Created a task [here](https://github.com/camunda/connectors-bundle/issues/680) so that we don't forget.

I'll fix it closer to release or when I have time. Alternatively, you can fix that too, and I can help you with testing.

Reply:
@chillleader Do we have to introduce this change?

Reply:
@chillleader ok, just maybe don't commit on much time. For me it will take ~2-3 hours.

Also, `camunda-platform` will also need this change reflected. https://github.com/camunda/camunda-platform/blob/main/docker-compose.yaml#L151

Reply:
I removed this port change altogether so that we don't have to change the Helm charts.